CharacterFactory.py

```diff
@@ -32,19 +35,19 @@ def PlotRisk(self, attacks: list) -> None:
         for attack in attacks:
             name, mech, magic, health = attack
             print(f"{name} ", end="")
-            if self.mechanical <= mech:
+            if self.maxMechanical <= mech:
                 # attack stuns opponent ?
                 print(risk_levels["high"])
                 continue
 
             total_damage = max(0, magic - self.magicDefense) + max(
                 0, health - self.defense
             )
-            if total_damage >= self.health:
+            if total_damage >= self.maxHealth:
                 print(risk_levels["high"])
-            elif total_damage > self.health / 2:
+            elif total_damage > self.maxHealth / 2:
                 print(risk_levels["medium"])
-            elif total_damage > self.health / 4:
+            elif total_damage > self.maxHealth / 4:
```

Comment:
@NoahBartell @instMeyFTW for the risk, should we care about the health or max health?